src/components/OSCALControlModification.js

```diff
@@ -42,13 +42,30 @@ function getAlterAddsOrRemovesDisplay(addsElements, addsLabel, controlPartId) {
         Name: {item.name}, Value: {item.value}
       </Typography>
     ));
+  let removedTypographies = null;
+  if (addsLabel === "Removes ") {
+    removedTypographies = addsElements
+      .flatMap((elt) => elt ?? [])
+      .map((y) => (
+        <Typography
+          color="textSecondary"
+          paragraph
+          variant="body1"
+          key={y.id}
+          style={{ textDecorationLine: "line-through" }}
+        >
+          Attribute: {Object.keys(y)[0]}, Value: {Object.values(y)[0]}
```

Comment:
Is there a reason that this is so different than how we do the `adds` on line 42?

https://github.com/EasyDynamics/oscal-react-library/blob/57b305bba845f4c6593549d57246f23248e8b8d8/src/components/OSCALControlModification.js#L42

I feel like it is a bit unclear what values we are actually displaying here. It may be that `adds` and `removes` are just very different in the `json`...

Reply:
Well we really have an object with multiple keys. In fact, I think the implementation of `adds` here is pretty basic and focuses purely on `props`; however using `adds` we can actually add anything. I thought we supported more but I really guess we don't.

The structure for an [`add`](https://pages.nist.gov/OSCAL/reference/latest/profile/json-reference/#/profile/modify/alters/removes) is actually very similar to that for [`remove`](https://pages.nist.gov/OSCAL/reference/latest/profile/json-reference/#/profile/modify/alters/removes).

So in that case, even `add` has more to support adding than what we're showing.

So we're supporting `removes` somewhat unequally here, by supporting removing more than just props. Except it's represented in a slightly confusing way because of unclear guidance in the ticket.

My expectation was that in supporting `removes` we would not just put a strike-through on text in the dialog; but that we would list `removes` where we cannot support strike-throughs (such as for props or links) and then we would use the strike-through for text in the area where we support viewing prose (so that you can see those modifications inline).

I am worried at this point that properly supporting this will require a more holistic rework of how we manage profiles, their resolution, and how the `OSCALControl` and other components operate.